### DISCUSSION TITLE
What is the natural language processing model for the text modal data processing of the AutoMM module in AutoGluon?

### DISCUSSION THREAD

**Discussion by imrooki** [autogluon/autogluon#2996] [Q&A] [answered]
What is the natural language processing model for the text modal data processing of the AutoMM module in AutoGluon？Is BERT? The Transformer in the Multimodal Network with Late Fusion is BERT?

**Comment by gradientsky** [comment, marked as answer]
Default may change between different presets and over time. Current defaults are available in the following config: https://github.com/autogluon/autogluon/blob/master/multimodal/src/autogluon/multimodal/presets.py#L85C19-L191

**Reply by imrooki** [reply]
Thanks
